### PR TITLE
(app) Improve UI

### PR DESCRIPTION
**Pull request by arpitBhalla**

### CODE REVIEW

File: packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/ActionList.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Text } from "@dyson/components/atoms/text/Text";
+import { css } from "@emotion/react";
+import { Conditional } from "@dyson/components/layouts";
+
+export const ActionList = ({ children, title }: { title?: string; children: React.ReactChild | React.ReactChild[] }): JSX.Element => {
+	return (
+		<div
+			css={css`
```
Comment (utkarsh-dixit) [inline packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/ActionList.tsx:9]: I think we can leave the margin to the parent component to set

File: packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	padding: 26rem;
+`;
+
+export function Steps(): JSX.Element {
+	const [s, setS] = React.useState([]);
+	const [selected, setSelected] = React.useState(new Set());
+
+	const toggleStep = React.useCallback(
+		(index) => {
+			const selectedSteps = new Set(selected);
```
Comment (utkarsh-dixit) [inline packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx:20]: Using index as a unique key is usually a bad idea, let's use a map here with a unique that out recorder will assign each step.

File: packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+				css={css`
+					overflow-y: auto;
+					padding: 18rem 26rem;
+					padding-top: 0rem;
+					height: 25vh;
+				`}
+			>
+				{[...Array(6)].map((e, i) => (
+					<Step
+						isSelectAllType={false}
```
Comment (utkarsh-dixit) [inline packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx:98]: We can remove this prop

File: packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+			`}
+		>
+			<div
+				css={css`
+					display: flex;
+					align-items: center;
+					padding: 14rem 26rem;
+				`}
+			>
+				<Checkbox isSelected={!!s.length} callback={(a) => setS([a])} />
```
Comment (utkarsh-dixit) [inline packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx:46]: Selecting this should select all the steps, and deselecting it should deselect all the steps. If the user selects all the steps manually it should automatically be selected

File: packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Action.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+			css={css`
+				padding: 26rem;
+				padding-top: 0rem;
+				height: 50vh;
+				overflow-y: auto;
+			`}
+		>
+			<ActionList>
+				<div
+					css={css`
```
Comment (utkarsh-dixit) [inline packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Action.tsx:38]: Move the css to a variable from inline css

**Review by utkarsh-dixit — CHANGES_REQUESTED** [review]
LGTM after these changes

File: packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx (hunk 2 above)
Comment (arpitBhalla) [reply packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx:20]: This is only for implementation, we can do it later on

File: packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx (hunk 3 above)
Comment (arpitBhalla) [reply packages/crusher-electron-app/src/extension/v2/ui/components/Sidebar/Steps.tsx:98]: there was type error

**Comment by utkarsh-dixit** [comment]
Merged in [0e2229f98256b3ebad8c80efcda464d2fb16bd9f](https://github.com/crusherdev/crusher/commit/0e2229f98256b3ebad8c80efcda464d2fb16bd9f)

**Pull request closed without merging**